### ISSUE TITLE
Trusted users and actions

### ISSUE THREAD

**Issue by gkatsev** [oftn/oftn-bot#53]
There should be a way to configure trusted users and actions.
These actions are only allowed to be performed by the trusted users.

**Comment by gkatsev** [comment]
Inviting the bot to a channel ( #52 ) can be a trusted action.

**Comment by eligrey** [comment]
Yeah I second this. Trusted users should be able to !trust/!untrust other users. There would need to be 2 tiers of trust though (one that can't be removed with this command), so we can keep board members permanently trusted.

**Comment by eligrey** [comment]
Also managing the [twitter mappings](https://github.com/oftn/oftn-bot/blob/b3ae1cca290631a7bc154ba540ec366e1c22ea13/oftnbot.js#L508) could be a trusted action. `!twitter-alias eligrey ^EG`

**Comment by brigand** [comment]
What's the story with saving things like this between restarts? If join commands are added, and trusted users are dynamic at runtime, doesn't there need to be a way to update the profile file, or store it in a db or something?

**Comment by ljharb** [comment]
if factoids.json can be updated, trustedUsers.json could be as well, no?
